src/EPR.RegulatorService.Facade.Core/Enums/RegistrationStatus.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace EPR.RegulatorService.Facade.Core.Enums;
+
+public enum RegistrationStatus
```

Comment:
Are you assuming the order here is the sort order when being displayed?

Reply:
No, they are just enums....